addons/mrp_subcontracting_purchase/tests/test_mrp_subcontracting_purchase.py

```diff
@@ -535,7 +537,7 @@ def test_subcontracting_lead_days_on_overview(self):
         bom_data = self.env['report.mrp.report_bom_structure']._get_bom_data(self.bom, self.warehouse, self.finished)
         self.assertEqual(bom_data['lead_time'], 15 + 5 + 5 + 0,
             "Lead time = Purchase lead time(finished) + Days to Purchase + Purchase security lead time + DTPMO on BOM")
-        self.assertEqual(bom_data['resupply_avail_delay'], 10 + 5 + 5 + 20,
+        self.assertEqual(bom_data['resupply_avail_delay'], 15 + 5 + 5 + 20,
```

Comment:
why ?

Reply:
OK, I think I figured this out :slightly_smiling_face: 

Normally:

```
resupply avail delay = 
resupply delay +
max(vendor lead time, manufacture lead time) +
max component delay
```

In Purchase, we also need to add to that "Purchase security lead days" and "Days to Purchase".

In this case:
* **resupply delay** = 0 (False in `_get_resupply_availability` in MRP, because route type is not "manufacture")
* **vendor lead time** = 15 (`delay` of the final product)
* **manufacture lead time** = 10 (`produce_delay` of the BoM)
* **Purchase security lead days** = 5 (`po_lead`)
* **Days to Purchase** = 5 (`days_to_purchase`)
* **max component delay** = 20

The only surprise for me is the last part, which I would expect to be 10 (we have 2 components, with delays 10 and 6). But then I realized that in Purchase, we also add `po_lead` and `days_to_purchase` to component delay :wink:  So it's `10 + 5 + 5 = 20`.

The final value is `0 + max(15, 10) + 5 + 5 + 20 = 15 + 5 + 5 + 20`, which is what the test checks after my changes :smile:  I will update the assertion below, so we don't have to investigate this again :wink: